GRAMMAR.md

```diff
@@ -53,12 +53,13 @@ unop ::= `-´ | not | `#´
 Modified rules from http://lua-users.org/lists/lua-l/2010-12/msg00699.html
 ```
 value ::= nil | false | true | Number | String | '...' | function |
-	tableconstructor | functioncall | var | '(' exp ')'
+	tableconstructor | anonymousfunction | functioncall | var | '(' exp ')'
 exp ::= unop exp | value [binop exp]
 prefix ::= '(' exp ')' | Name
 index ::= '[' exp ']' | '.' Name
 call ::= args | ':' Name args
 suffix ::= call | index
 var ::= prefix {suffix} index | Name
+anonymousfunction ::= function funcbody
```

Comment:
This grammar file already came from Lua, we don't need to modify it to add features I don't think?

Reply:
Didn't know that, I'll remove it.